app.js

```diff
@@ -59,10 +59,10 @@ function runNpm(command) {
 try {
 	require('sugar');
 } catch (e) {
-	runNpm('install');
+	runNpm('install --production');
```

Comment:
The command is interpolated in `runNpm` function, and yes, it does work, I even removed `node_modules` to test.